outofcore/include/pcl/outofcore/cJSON.h

```diff
@@ -44,7 +44,7 @@ extern "C"
 #define cJSON_IsReference 256
 
 /* The cJSON structure: */
-typedef struct cJSON {
+typedef struct cJSON {	// NOLINT
```

Comment:
Why did you decide to turn linting off here? Does something like `using cJSON = struct { ... };` not work? (same below for cJSON_Hooks)

Reply:
I'll look again (I have to reinstall my Ubuntu machine), but I believe it's in an `extern C` section

Reply:
You're right, it is, but `using` should still be possible. But anyway, I'm okay with keeping `typedef` and adding `NOLINT`